jenkins-design-language/src/js/components/status/SvgSpinner.jsx

```diff
@@ -6,6 +6,33 @@ import { describeArcAsPath } from '../SVG';
 export const strokeWidth = 3.5; // px. Maybe we can fetch this from CSS at runtime in the future
 
 export default class SvgSpinner extends Component {
+    componentWillMount() {
+        this.infiniteRotationRunning = false;
+        this.setState({
+            infiniteRotatePercentage: 0
+        });
+    }
+
+    infiniteLoadingTimer = () => {
+        let infiniteRotatePercentage = this.state.infiniteRotatePercentage;
+        
+        infiniteRotatePercentage += 1.5;
+
+        if (infiniteRotatePercentage >= 360) {
+            infiniteRotatePercentage = 0;
```

Comment:
... and instead of `= 0` here, do `infiniteRotatePercentage -= 360`.

Also if it's degrees and not percentage we might want to name it differently.